prepare_zephir_records/integration_tests/how_to_run_integration_test.md

```diff
@@ -0,0 +1,5 @@
+# How to run the integration tests
+
+```
+./run_integration_tests.sh <password_for_htmmdba>"
```

Comment:
In the environment where this is run, is it fine for the htmmdba password to be saved in the command line history? Would providing it via stdin be preferable?

Reply:
We should maybe use a locked down config file in the format mysql expects. See second, preferred answer: https://unix.stackexchange.com/questions/205180/how-to-pass-password-to-mysql-command-line

Reply:
Thank you both for the suggestions. I will try the config file approach after merging these changes.
Jing